README.md

```diff
@@ -135,21 +192,28 @@ Released under the BSD-3-Clause License. See `LICENSE` for more information.
+- [Apache Commons Collections](https://commons.apache.org/proper/commons-collections/) for enhanced collections under [Apache v2](https://www.apache.org/licenses/).
+- [Apache Commons CSV](https://commons.apache.org/proper/commons-csv/) for CSV handling under [Apache v2](https://www.apache.org/licenses/).
+- [Apache SIS](https://sis.apache.org/) for coordinate system transformations and COG support under [Apache v2](https://www.apache.org/licenses/).
 - [Checkstyle](https://checkstyle.sourceforge.io) under [LGPL](https://github.com/checkstyle/checkstyle/blob/master/LICENSE).
 - [D3](https://d3js.org/) for data visualization under [ISC](https://github.com/d3/d3/blob/main/LICENSE).
-- [ESLint](https://eslint.org/) for code style enforcement under [MIT](https://github.com/eslint/eslint/blob/main/LICENSE).
+- [GeoTools](https://geotools.org/) for geospatial data processing under [LGPL](https://github.com/geotools/geotools/blob/main/LICENSE.md).
 - [Gradle](https://gradle.org) under [Apache v2](https://github.com/gradle/gradle?tab=Apache-2.0-1-ov-file#readme).
-- [HTTP Components](https://hc.apache.org/httpcomponents-client-5.4.x/index.html) under [Apache v2](https://www.apache.org/licenses/).
-- [Jetty](https://jetty.org/index.html) under [Apache v2](https://jetty.org/docs/jetty/12/index.html).
+- [JTS Topology Suite](https://locationtech.github.io/jts/) for geometry handling under [EDL](https://www.eclipse.org/org/documents/edl-v10.php).
 - [JUnit](https://junit.org/junit5/) under [EPL v2](https://github.com/junit-team/junit5).
+- [Math.js](https://mathjs.org/) for mathematical expressions under [Apache v2](https://github.com/josdejong/mathjs/blob/develop/LICENSE).
 - [Minio Java SDK](https://min.io/docs/minio/linux/developers/java/minio-java.html) under [Apache v2](https://github.com/minio/minio-java?tab=Apache-2.0-1-ov-file#readme).
+- [Mockito](https://site.mockito.org/) for testing under [MIT](https://github.com/mockito/mockito/blob/main/LICENSE).
+- [Picocli](https://picocli.info/) for command line parsing under [Apache v2](https://github.com/remkop/picocli/blob/main/LICENSE).
+- [Popper.js](https://popper.js.org/) for tooltip positioning under [MIT](https://github.com/floating-ui/floating-ui/blob/master/LICENSE).
 - [Public Sans](https://public-sans.digital.gov/) font under [OFL-1.1](https://github.com/uswds/public-sans/blob/master/LICENSE.md).
-- [QUnit](https://qunitjs.com/) for unit front-end testing under [MIT](https://github.com/qunitjs/qunit/blob/main/LICENSE.txt).
-- [SVG Spinners](https://github.com/n3r4zzurr0/svg-spinners?tab=readme-ov-file) under [MIT](https://github.com/n3r4zzurr0/svg-spinners?tab=readme-ov-file)
+- [SLF4J](https://www.slf4j.org/) for logging under [MIT](https://github.com/qos-ch/slf4j/blob/master/LICENSE.txt).
+- [Spotless](https://github.com/diffplug/spotless) for code formatting under [Apache v2](https://github.com/diffplug/spotless/blob/main/LICENSE).
```

Comment:
This is under Apache v2 but please correct this URL to https://github.com/diffplug/spotless/blob/main/LICENSE.txt